vocabularies/Analytics.json

```diff
@@ -161,6 +164,114 @@
+            "@Core.LongDescription": "Each element in the collection defines the properties that constitute one level.\n            A property must not referenced by more than one level.\n            The first element in the collection defines the property names of the coarsest level,\n            the following elements define the property names of consecutively finer-grained aggregation levels.\n            The function result is the leveled hierarchy with these levels in preorder,\n            entries on the finest-grained level cannot be expanded further.\n            All referenced properties must be groupable."
+          },
+          {
+            "$Name": "Aggregation",
+            "$Collection": true,
+            "@Core.Description": "Properties to aggregate for all result entries on all levels",
+            "@Core.LongDescription": "All properties in this collection must be custom aggregates."
+          },
+          {
+            "$Name": "SiblingOrder",
+            "$Collection": true,
+            "$Type": "Analytics.MultiLevelExpandSiblingOrder",
+            "@Core.Description": "Sort specification to apply to all direct descendants of a given entry in the resulting leveled hierarchy"
+          },
+          {
+            "$Name": "ShowLevels",
+            "$Type": "Edm.Int64",
+            "@Core.Description": "Number of levels to be shown in the initial expansion (absent means all levels)",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "ExpandEntries",
+            "$Collection": true,
+            "$Type": "Analytics.MultiLevelExpandEntry",
+            "@Core.Description": "Entries with exceptional expansion",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "SubtotalsAtBottom",
+            "$Type": "Edm.Bool",
+            "@Core.Description": "Whether to duplicate the group headers so that they appear before and after their descendants",
+            "@Core.LongDescription": "The entry before has [DrillState](Hierarchy.md#HierarchyType) `expanded`,\n            the entry after has DrillState `subtotal`.",
+            "@Core.OptionalParameter": { "DefaultValue": "false" }
+          }
+        ],
+        "$ReturnType": {
+          "$Collection": true,
+          "$Type": "Edm.EntityType",
+          "@Core.Description": "Output set including the instance annotation [`LevelInformation`](#LevelInformation)"
+        }
+      }
+    ],
+    "MultiLevelExpandLevel": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "@Core.Description": "Property names constituting a level in an [unnamed leveled hierarchy](#MultiLevelExpand)",
+      "@Core.LongDescription": "Properties in `D` must be used to identify entries in [`ExpandEntries/Entry`](#MultiLevelExpandEntry),\n          otherwise they have the same effect as properties in `A`.",
+      "D": {
```

Comment:
"D" and "A" are unusually short names. Why not "DimensionProperties" and "AdditionalDimensionProperties"?

Reply:
They are so technical that I do not think longer names make anything clearer. And the properties in A must also appear in D, therefore "AdditionalDimensionProperties" would be misleading.